webapp/channels/src/packages/mattermost-redux/src/selectors/entities/posts.ts

```diff
@@ -800,3 +802,10 @@ export function makeGetPostAcknowledgementsWithProfiles(): (state: GlobalState,
         },
     );
 }
+
+export function getTeamIdFromPost(state: GlobalState, postId: Post['id']): Team['id']|undefined {
```

Comment:
Instead of having this sometimes return undefined, I think we might want to have it default to `''` if the channel isn't found to avoid having to fight with typing wherever this is used

Reply:
@hmhealey the issue I see with this is if the channel isn't found it's gonna assume it belongs in the current team.

Should we expect all channels the user is a member of to be available at all times?

Reply:
Yeah, we can safely assume that. That's been the case since the cross-team channel switcher was added.

I could see potentially needing to change that in the future, but I don't think we need to worry about that for now